Improve instance dump for `jail`

AlexeyAndKartashov opened the pull request:
`mount` and `setns` properties apply exclusively to `jail` data.
But in instance dump they are displayed as if they relate directly to the instance properties:

```sh
$ ubus call service list '{"name": "sysntpd"}'
{
        "sysntpd": {
                "instances": {
                        "instance1": {
                                "running": true,
                                "pid": 7405,
                                "command": [
                                        "/usr/sbin/ntpd",
                                        "-n",
                                        "-N",
                                        "-S",
                                        "/usr/sbin/ntpd-hotplug",
                                        "-p",
                                        "0.openwrt.pool.ntp.org",
                                        "-p",
                                        "1.openwrt.pool.ntp.org",
                                        "-p",
                                        "2.openwrt.pool.ntp.org",
                                        "-p",
                                        "3.openwrt.pool.ntp.org"
                                ],
                                "term_timeout": 5,
                                "respawn": {
                                        "threshold": 3600,
                                        "timeout": 5,
                                        "retry": 5
                                },
                                "no_new_privs": true,
                                "capabilities": "/etc/capabilities/ntpd.json",
                                "user": "ntp",
                                "group": "ntp",
                                "jail": {
                                        "name": "ntpd",
                                        "procfs": false,
                                        "sysfs": false,
                                        "ubus": true,
                                        "log": false,
                                        "ronly": false,
                                        "netns": false,
                                        "userns": false,
                                        "cgroupsns": false,
                                        "console": false
                                },
                                "mount": {
                                        "/bin/ubus": "0",
                                        "/usr/bin/env": "0",
                                        "/usr/bin/jshn": "0",
                                        "/usr/sbin/ntpd-hotplug": "0",
                                        "/usr/share/libubox/jshn.sh": "0"
                                }
                        }
                }
        }
}

aparcar commented:
@dangowrt do you mind fixing it while committing? would be a bummer not to have this patch

AlexeyAndKartashov committed:
instance: fix instance dump

Move the `mount` and `setns` from instance data into the `jail` object

Signed-off-by: AlexeyAndKartashov <alexey.and.kartashov@gmail.com>

AlexeyAndKartashov committed:
Improve instance dump for jail data

`mount` and `setns` properties apply exclusively to jail data.

But in instance dump they are displayed as if they relate directly to the instance properties

AlexeyAndKartashov commented:
@dangowrt Sorry, I completely forgot about your request to correct the commit. Done

dangowrt commented:
Please add your `Signed-off-by:`-tag,, and also avoid lines longer than 70 characters in commit message.
